api/DHSC.FingertipsNext.Modules.Core.Service/HealthMeasureService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DHSC.FingertipsNext.Modules.Core.Repository;
+using DHSC.FingertipsNext.Modules.Repository;
+using Microsoft.EntityFrameworkCore;
+
+namespace DHSC.FingertipsNext.Modules.Core.Service
+{
+    public sealed class HealthMeasureService : IHealthMeasureService
+    {
+        private readonly RepositoryDbContext dbContext;
+
+        public HealthMeasureService(RepositoryDbContext repositoryDbContext)
+        {
+            this.dbContext = repositoryDbContext ?? throw new ArgumentNullException(nameof(repositoryDbContext));
+        }
+
+        public async Task<HealthMeasure> GetFirstHealthMeasure()
+        {
+            var healthMeasures = await dbContext.HealthMeasures.ToListAsync();
```

Comment:
By converting to a List before taking the first, it will essentially do a SELECT * and take the first element in code using Linq, rather than doing a SELECT TOP 1 * against the database.

Reply:
This is a placeholder to go with the placeholder controller - We can discuss what to do here when we discuss the controller mentioned below.